plugins/community/infobloxopen-gorm/v1.1.2/Dockerfile

```
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:1.19.2-bullseye AS build
```

```suggestion
FROM golang:1.19.4-bullseye AS build
```

When creating new plugins, we update to the latest base images.

---

Updated to new version